src/responder/kcm/kcmsrv_ops.c

```diff
@@ -2298,8 +2417,10 @@ static struct kcm_op kcm_optable[] = {
 };
 
 /* MIT EXTENSIONS. */
-#define KCM_MIT_OFFSET 13000
+#define KCM_MIT_OFFSET 13001
```

Comment:
Can we use `KCM_OP_MIT_EXTENSION_BASE` from libkrb5? Or is it private there?

Reply:
It is private.

Reply:
How are lib users expected to match op-code with operation required?

How is it done for "general" (not "MIT extensions") ops?

Reply:
What do you mean?

Reply:
Do we rely on the fact order of opcodes in our `kcm_optable` is the same as in libkrb5?
(For new opcodes this is order + value of KCM_MIT_OFFSET.(

Reply:
Yes.

Reply:
This feels error-prone.

Reply:
This is not something that's going to change, like ever.

Reply:
> Do we rely on the fact order of opcodes in our kcm_optable is the same as in libkrb5? 

KCM isn't an krb5-defined format.  opcodes must match across all implementations for them to interoperate, and there's no published standard.